.gitignore

```diff
@@ -28,3 +28,9 @@ app/database_etl/tableau_data_connector/token.pickle
 
 # cached Stan models
 **/stanmodelcache*
+.vscode
+
+# CSVs folder used to sync dbs
+csvs/
+load-dotenv.sh
+push_csvs.py
```

Comment:
Unrelated - I wonder if we should commit this script, we're likely to have to use it in the future

Reply:
I agree that we should! Just wasn't sure that it should be in the parent folder or where the right spot to put it was.